frontend-react/src/AppRouter.tsx

```diff
@@ -33,6 +33,13 @@ import { AdminRevHistoryWithAuth } from "./pages/admin/AdminRevHistory";
 import { ErrorNoPage } from "./pages/error/legacy-content/ErrorNoPage";
 import { MessageDetailsWithAuth } from "./components/MessageTracker/MessageDetails";
 
+export enum FeatureType {
```

Comment:
Just a nit, non-blocking: to keep with the `EventName` nomenclature, could this be `FeatureName`?